python/test_downloader.py

```
@@ -2476,6 +2502,77 @@ def download(self, _urls):
+            ),
+        ):
+            with patch.object(downloader, "_get_temp_dir", return_value=temp_dir):
+                with patch.object(downloader, "_get_cached_video_path", return_value=None):
+                    with patch.object(
+                        downloader,
+                        "_find_downloaded_file",
+                        return_value=downloaded_file,
+                    ):
+                        with patch.object(downloader, "_check_file_stability", return_value=True):
+                            with patch("downloader.shutil.copy2") as mock_copy:
+                                mock_copy.side_effect = lambda src, dst: Path(dst).write_bytes(b"copied data")
```

Ruff (0.15.1)

[warning] Unused lambda argument: `src`

(ARG005)